<launch>
  <node name="culvert_sim" pkg="culvert_sim" type="indoor_test.py" output="screen">
        <param name="w" value="1.5" />
        <param name="l" value="5.0" />
        <param name="h" value="0.8" />
        <param name="scale" value="10" />
        <param name="offset_x" value="0.0" />
        <param name="offset_y" value="-0.587" />
        <param name="offset_z" value="0.0" />
        <param name="h_fov" value="62.0" />
        <param name="v_fov" value="48.5" />
        <param name="near" value="0.2" />
        <param name="far" value="3.0" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find culvert_sim)/rviz/rviz_indoor.rviz" />
</launch>
